i18n/de/docusaurus-plugin-content-docs/current/help-center/your-account/account-settings.mdx

```diff
@@ -488,6 +488,31 @@ Abhängig davon in welchem Status sich das Chat-Backup befindet, haben Sie zwei
 
 In beiden Fällen, stellen Sie sicher, dass Sie die erstellte Passphrase sicher aufheben.
 
+## Chat-Einstellungen
+
+In diesem Abschnitt können Sie die Aktion „Drücken Sie die Eingabetaste“ ändern.
```

Comment:
Same as for the English version, would add a sub-headline that says that this is for changing the function of the enter key, e.g. "Verhalten der Eingabetaste ändern"

Also would add information that this setting is per device.
I just tested what happens when I log out after changing the setting and log in to another account and it was persisted. Probably we would need to mention this for users with shared devices? Or probably this isn't the intended behaviour?